conans/client/graph/proxy.py

```diff
@@ -13,6 +13,9 @@
 from conans.util.tracer import log_recipe_got_from_local_cache
 
 
+DEPRECATED_CONAN_CENTER_BINTRAY_URL = "https://conan.bintray.com"
```

Comment:
I'd add a TODO note asking to remove this code when conan.bintray.com is definitely shut down

Reply:
```suggestion
# TODO: Remove this message when the url is no longer available
DEPRECATED_CONAN_CENTER_BINTRAY_URL = "https://conan.bintray.com"
```